packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebox/_index.scss

```diff
@@ -0,0 +1,9 @@
+.dx-chat-message-box {
```

Comment:
Scenario is not considered
<img width="126" alt="image" src="https://github.com/user-attachments/assets/0d2e8f91-fc4b-4d6a-905b-61ae7333020b">

Reply:
In discussion with the designer, we decided that we would choose the minimum chat width later. In addition, in yesterday's discussion we agreed not to think about responsiveness now. I have created a task about it.